lib/simple_url_preview.dart

```diff
@@ -86,7 +90,9 @@ class _SimpleUrlPreviewState extends State<SimpleUrlPreview> {
   int? _descriptionLines;
   TextStyle? _siteNameStyle;
   Color? _imageLoaderColor;
+  
```

Comment:
Can you remove this extra line? Overall PR looks good. Thanks for your contribution!

Reply:
Sure, thanks